faq.qmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Frequently Asked Questions"
+format:
+  html:
+    toc: true
+---
+
+### Question: Are you part of (Posit, Google, etc)? {#independent}
+
+Answer: No! 
```

Comment:
may consider https://quarto.org/docs/authoring/callouts.html for extra emphasis later on? or formatting as collapsible callouts? not important for v1

Reply:
Yeah, I figure there's probably a FAQ plugin or something somewhere. For now I went with "good enough".